docs/content/latest/benchmark/ycsb.md

```diff
@@ -24,161 +24,24 @@ For more information about YCSB see:
-
-```sh
-cd $HOME
-git clone https://github.com/yugabyte/cqlsh
-```
-
-## Step 5. Prepare the driver script
-
-1. Create an executable file in the YCSB folder:
-
-```sh
-cd $HOME/YCSB
-touch run-yb-ycsb.sh
-chmod a+x run-yb-ycsb.sh
-```
-
-2. Copy the following contents in `run-yb-ycsb.sh`:
-
-{{< note title="Note" >}}
-
-You may want to customize the values below (such as <ip-addr>) with the correct/intended ones for your setup.
-
-{{< /note >}}
-
-```sh
-#!/bin/bash
-
-
-# YB-CQL host (any of the yb-tserver hosts)
-# (The other nodes should get automatically discovered by the driver).
-hosts=127.0.0.1
-ycsb=$HOME/YCSB/bin/ycsb
-cqlsh=$HOME/cqlsh/bin/cqlsh <ip-addr>
-ycsb_setup_script=$HOME/YCSB/cassandra/src/test/resources/ycsb.cql
-keyspace=ycsb
-table=usertable
-# See https://github.com/brianfrankcooper/YCSB/wiki/Core-Properties for param descriptions
-params="-p recordcount=1000000 -p operationcount=10000000"
-
-setup() {
-    $cqlsh <<EOF
-create keyspace $keyspace with replication = {'class': 'SimpleStrategy', 'replication_factor' : 3};
-EOF
-    $cqlsh -k $keyspace -f $ycsb_setup_script
-}
-
-cleanup() {
-    $cqlsh <<EOF
-drop table $keyspace.$table;
-drop keyspace $keyspace;
-EOF
-}
-
-delete_data() {
-    $cqlsh -k $keyspace <<EOF
-drop table $table;
-EOF
-    $cqlsh -k $keyspace -f $ycsb_setup_script
-}
-
-
-run_workload() {
-    local workload=$1-----------------
-    echo =========================== $workload ===========================
-    $ycsb load cassandra-cql -p hosts=$hosts -P workloads/$workload $params \
-      -p threadcount=40 | tee $workload-load.dat
-    $ycsb run cassandra-cql -p hosts=$hosts -P workloads/$workload $params \
-      -p cassandra.readconsistencylevel=QUORUM -p cassandra.writeconsistencylevel=QUORUM \
-      -p threadcount=256 -p maxexecutiontime=180 | tee $workload-transaction.dat
-    delete_data
-}
-
-setup
-
-load_data workloada
-run_workload workloada
-run_workload workloadb
-run_workload workloadc
-run_workload workloadf
-run_workload workloadd
-
-truncate_table
-load_data workloade
-run_workload workloade
-
-cleanup
-```
-
-We use YugabyteDB with strongly consistent reads and writes, which corresponds, in Cassandra, to using the `QUORUM` option for both `cassandra.readconsistencylevel` and `cassandra.writeconsistencylevel` (see the command above).
-
-## Step 6. Run and check results
-
-Simply run the script above:
-
-```sh
-./run-yb-ycsb.sh
-```
-
-{{< tip title="Tip" >}}
-
-See also [this page](https://github.com/brianfrankcooper/YCSB/wiki/Running-a-Workload-in-Parallel) about running workloads in parallel.
-
-{{< /tip >}}
-
-Results for each workload will be in `workload[abcdef]-transaction.dat` (e.g. `workloada-transaction.dat`).
+<ul class="nav nav-tabs nav-tabs-yb">
+  <li >
+    <a href="#yugabytecql" class="nav-link active" id="yugabytecql-tab" data-toggle="tab" role="tab" aria-controls="yugabytecql" aria-selected="true">
+      yugabyteCQL
```

Comment:
Most of our docs just use `YCQL` and `YSQL` instead of `yugabyteCQL` and `yugabyteSQL`.

Reply:
It was Karthik and Mihnea that suggested the use of yugabyteSQL and yugabyteCQL while committing the benchmark code.
I hope to retain them, as it will be easier for people.

Let me know if you still want me to change it.

Reply:
That comment was for folder names in the repo. All our docs have tabs named YCQL/YSQL, for example:
https://docs.yugabyte.com/latest/secure/authorization/rbac-model/
Lets keep it consistent with the rest of the docs.

Reply:
Done.